src/components/Routes/Registration.tsx

```diff
@@ -109,13 +132,31 @@ const Registration = ({ navigation }: Props) => {
         }
       })
       .catch(
-        handleFormSubmitError<RegistrationFlow | undefined>(
-          setConfig,
+        handleFormSubmitError(
```

Comment:
What's confusing here I guess is that OIDC looks like an error but it's actually intended. I think we generally need to refactor this code and maybe offer it as part of Ory Elements.

Reply:
Yes, but this code will be in the language of the native app SDK, so it would not be enough to suppy a JS/TS implementation.